builtin/k8s/platform.go

```diff
@@ -771,29 +839,6 @@ func (p *Platform) resourceDeploymentCreate(
 		return false, nil
 	})
 	if err != nil {
-		step.Update("Error detected waiting for Deployment to start.")
-		step.Status(terminal.StatusError)
-		step.Abort()
-
-		ui.Output("The following is events for pods observed while attempting to start the Deployment", terminal.WithWarningStyle())
-
-		for uid, name := range podsSeen {
-			sel := ev.GetFieldSelector(nil, nil, nil, (*string)(&uid))
-
-			events, err := ev.List(ctx, metav1.ListOptions{
-				FieldSelector: sel.String(),
-			})
-			if err == nil {
-				ui.Output("Events for %s", name, terminal.WithHeaderStyle())
-				for _, ev := range events.Items {
-					if ev.Type == "Normal" {
-						continue
-					}
-					ui.Output("  %s: %s (%s)", ev.Type, ev.Message, ev.Reason)
-				}
-			}
-		}
-
```

Comment:
Hm, I think all of this (686, then 705 to here) was accidentally deleted? Same with the following diff with `deleteGrace` and deleting gracefully in the destroy func.